.github/workflows/run-snapshot-test.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Why this is added?

Reply:
One of the linters has a warning that "File is not started with ---" , so I assumed that it is a standard way for .yaml files.